webpack.common.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
           {
             loader: 'css-loader',
             options: {
-              url: false, sourceMap: true
+              modules: false, url: false, sourceMap: true
```

Comment:
In the updated, css-loader CSS modules are not enabled by default for speed.  The default is echoed here as a reminder in case we start using CSS modules.